src/MassTransit/Internals/Json/InternalJsonExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿#nullable enable
+
+using MassTransit.Metadata;
+using System;
+using System.Text.Json;
+
+namespace MassTransit.Internals.Json
```

Comment:
These are just extensions, and can be in the Internals/Extensions folder, in the Internals namespace. With v8, we stopped using so many extensively nested namespaces as they just lead to clutter in the using statements.

Reply:
I have moved the file into its parent folder, and updated its namespace to `MassTransit.Internals`.